pandas/_libs/groupby.pyx

```diff
@@ -60,6 +67,14 @@ cdef enum InterpolationEnumType:
     INTERPOLATION_NEAREST,
     INTERPOLATION_MIDPOINT
 
+cdef inline bint check_inf(numeric_t ai) nogil:
```

Comment:
hmm i think we have this file somewhere? if not this should go in missing.pyx

Reply:
cnp.isinf i think?

Reply:
nope, cnp.isinf is actually not defined. Trying to build with it always results in an error:
```
pandas/_libs/groupby.pyx:275:48: cimported module has no attribute 'isinf'
```
Though there is such a function in numpy: https://github.com/numpy/numpy/blob/main/numpy/core/include/numpy/npy_math.h#L256. Maybe I just need to import it differently

Reply:
try `from numpy.math cimport isinf`